packages/@vue/cli-plugin-unit-jest/__tests__/jestPlugin.spec.js

```diff
@@ -137,5 +137,24 @@ test('should work with Vue 3', async () => {
   })
   const pkg = JSON.parse(await project.read('package.json'))
   expect(pkg.devDependencies['@vue/test-utils']).toMatch('^2')
+  expect(pkg.devDependencies['@vue/vue3-jest']).toMatch('^27')
```

Comment:
Please remove the Vue 3 tests in this file. They should be placed in the `jestPluginVue3.spec.js` so that they'll have the correct test environment setup. I think I forgot to remove this test case and it's a surprise that it still passed…

Reply:
Alright I removed them (I already had the same tests in `jestPlluginVue3.spec.js`), and CI is green, thanks.

I don't see `jestPluginVue3.spec.js` on the CI though? Is it running?

Reply:
Cool! It's running in the `e2e` job: https://app.circleci.com/pipelines/github/vuejs/vue-cli/1150/workflows/da9c2263-d7d7-4b0e-8f91-4d1c90303cae/jobs/27395

Reply:
Awesome then, it should be working.

The PR is ready for your review!